### PR TITLE
edit user

### PR DESCRIPTION
**Pull request by hieupc-0248**
## Related Tickets:
- [#68433](https://edu-redmine.sun-asterisk.vn/issues/68433)

## Notes Impacted Areas:
- app/Http/Controllers/UserController.php
- resources/views/users/edit.blade.php
- app/Http/Requests/UpdateUserRequest.php

## Attachments:
- 
image

### CODE REVIEW

**Comment by hieupc-0248** [comment]
ready

File: app/Http/Controllers/BookController.php [hunk 1]
```diff
@@ -104,8 +105,8 @@ public function edit(Book $book)
     /**
      * Update the specified resource in storage.
      *
-     * @param  \Illuminate\Http\Request  $request
-     * @param  \App\Models\Book  $book
+     * @param  \Illuminate\Http\Request $request
+     * @param  \App\Models\Book         $book
```
Comment (ghost) [inline app/Http/Controllers/BookController.php]:
```suggestion
     * @param  \App\Models\Book $book
```

File: resources/views/users/edit.blade.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                <form method="POST" action="{{ route('users.update', $user) }}" enctype="multipart/form-data">
+                    @csrf
+                    @method('PUT')
+
+                    <div class="mb-4">
+                        <label for="first_name" class="block font-bold mb-2">{{ __('First Name') }}</label>
+                        <input id="first_name" type="text" class="form-input rounded-lg w-full @error('first_name') border-red-500 @enderror" name="first_name" value="{{ old('first_name', $user->first_name) }}" required autocomplete="first_name" autofocus>
+                            @error('first_name')
+                            <p class="text-red-500 text-xs italic">{{ $message }}</p>
+                            @enderror
```
Comment (ghost) [inline resources/views/users/edit.blade.php]: indent 4 spaces

File: resources/views/users/edit.blade.php [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                    </div>
+
+                    <div class="mb-4">
+                        <label for="last_name" class="block font-bold mb-2">{{ __('Last Name') }}</label>
+                        <input id="last_name" type="text" class="form-input rounded-lg w-full @error('last_name') border-red-500 @enderror" name="last_name" value="{{ old('last_name', $user->last_name) }}" required autocomplete="last_name" autofocus>
+                            @error('last_name')
+                            <p class="text-red-500 text-xs italic">{{ $message }}</p>
+                            @enderror
```
Comment (ghost) [inline resources/views/users/edit.blade.php]: indent 4 spaces

File: resources/views/users/edit.blade.php [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+                    </div>
+
+                    <div class="mb-4">
+                        <label for="email" class="block font-bold mb-2">{{ __('Email') }}</label>
+                        <input id="email" type="text" class="form-input rounded-lg w-full @error('email') border-red-500 @enderror" name="email" value="{{ old('email', $user->email) }}" required autocomplete="email" autofocus>
+                            @error('email')
+                            <p class="text-red-500 text-xs italic">{{ $message }}</p>
+                            @enderror
```
Comment (ghost) [inline resources/views/users/edit.blade.php]: indent 4 spaces

File: resources/views/users/edit.blade.php [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+                    <div class="flex items-center justify-center w-full bg-blue-500 hover:bg-blue-700 h-10 rounded-lg">
+                        <button type="submit" class="btn btn-primary">
+                            {{ __('Save Changes') }}
+                        </button>
+                    </div>
+                </form>
+            </div>
+        </div>
+    </div>
+</x-app-layout>
```
Comment (ghost) [inline resources/views/users/edit.blade.php]: eof

**Comment by hieupc-0248** [comment]
ready

**Pull request merged**